src/Application/FileProcessor.php

```diff
@@ -143,9 +149,10 @@ private function printFile(File $file, Configuration $configuration, string $fil
              * On very first content level
              */
             $originalFileContent = $file->getOriginalFileContent();
-            $ltrimOriginalFileContent = ltrim($originalFileContent);
+            $ltrimOriginalFileContent = preg_replace(self::OPEN_TAG_SPACED_REGEX, '<?php', ltrim($originalFileContent));
+            $ltrimNewContent = preg_replace(self::OPEN_TAG_SPACED_REGEX, '<?php', ltrim($newContent));
```

Comment:
you can use `\Nette\Utils\Strings::replace($subject, $pattern, $replacement)` usage for consistenty, you can search how `Strings::replace(` usage on codebase

Reply:
Ah sure, switching it to this now thanks